tests/tuxemon/test_path_controller.py

```diff
@@ -38,6 +38,9 @@ def set_position(self, pos):
         self.tile_pos = pos
         self.position = Vector2(float(pos[0]), float(pos[1]))
 
+    def set_tile_position(self, pos):
+        self.tile_pos = pos
+
```

Comment:
`set_position` is also used to set `tile_pos`, but it also modifies the `position` attribute. Any risk of redundancy between the methods? Also, perhaps refactoring `set_position` to directly call `set_tile_position()` be also in order?